Documentation/design.md

```diff
@@ -120,7 +120,7 @@ The `Probe` custom resource definition (CRD) allows to declarative define how gr
 
 The `PrometheusRule` custom resource definition (CRD) declaratively defines desired Prometheus rules to be consumed by Prometheus or Thanos Ruler instances.
 
-Alerts and recording rules are reconciled by the Operatorand dynamically loaded without requiring any restart of Prometheus/Thanos Rulre.
+Alerts and recording rules are reconciled by the Operator and dynamically loaded without requiring any restart of Prometheus/Thanos Rulre.
```

Comment:
Thanks for the fix :)
Could you please also update `Thanos Rulre` to `Thanos Ruler`